src/woocommerce/style-my-account.scss

```diff
@@ -1,4 +1,4 @@
-@import "../shared/style";
+@use "../shared/style";
```

Comment:
Cool, could we also migrate [these @import,](https://github.com/search?q=repo%3AAutomattic%2Fgravatar-enhanced%20%40import%20%22..%2Fshared%2Fstyle%22%3B&type=code) thanks!